src/components/Dropdown/MenuItemContent/MenuItemContent.tsx

```diff
@@ -0,0 +1,84 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { IconProps, IconSize } from "@elements/Icon/Icon";
+import { ReactComponent as CaretRightIcon } from "@elements/Icon/Svg/CaretRight.svg";
+import { ReactComponent as CheckIcon } from "@elements/Icon/Svg/Check.svg";
+import { Size, Style } from "@utilities/enum";
+import { ReactElement } from "react";
+
+export enum SelectionIndicatorIcon {
+    Check = "Check",
+    CaretRight = "CaretRight",
+}
+
+type MenuItemContentBase = {
+    id: string;
+    title: string;
+    icon?: ReactElement<IconProps>;
+    style?: Style.Primary | Style.Danger;
+    disabled?: boolean;
+    active?: boolean;
+    subtitle?: string;
+    size?: Size.Small | Size.Large;
+    selectionIndicator?: SelectionIndicatorIcon;
+};
+
+export type MenuItem =
+    | (MenuItemContentBase & {
+          size: Size.Small;
+      })
+    | (MenuItemContentBase & {
+          subtitle: string;
+          size: Size.Large;
+      });
+
+export type MenuItemContentProps = Omit<MenuItem, "id">;
+
+export default function MenuItemContent({
+    title,
+    icon,
+    subtitle = "",
```

Comment:
not sure if we should set it empty, since it always will render the subtitle, or is that on purpose?